lib/accordion/index.js

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
```

Comment:
Move it to near the lines for which this needs to be disabled / resolve the issue.

Reply:
done !